qa/pages/SiteMapPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from 'next'
+import Link from 'next/link'
+import { NextSeo } from 'next-seo'
+import { ReactElement, useCallback } from 'react'
+import { FieldError, useForm } from 'react-hook-form'
+
+import { Container } from '@/components/Container/Container'
+import { RootLayout } from '@/components/Layout/RootLayout'
+import { TEXTAREA_MAX_CHARACTERS } from '@/constants/forms'
+import { useFormErrorHydration } from '@/hooks/useFormErrorHydration'
+import { logger } from '@/lib/logger'
+import { getValuesFromSearchParams } from '@/utils/form.utils'
+import { getCookieBanner } from '@/utils/getCookieBanner'
+import { FeedbackInputs, feedbackSchema } from '@/utils/schemas/feedback.schema'
+
+export type FeedbackProps = InferGetServerSidePropsType<typeof getServerSideProps>
+
+export default function Feedback({ query }: FeedbackProps) {
+  const { register, formState, setError, watch, handleSubmit } = useForm<FeedbackInputs>({
+    resolver: zodResolver(feedbackSchema),
+    defaultValues: getValuesFromSearchParams(feedbackSchema, query),
+  })
+
+  const handleFoundError = useCallback(
+    (field: keyof FeedbackInputs, error: FieldError) => setError(field, error),
+    [setError]
+  )
+
+  const { errors } = useFormErrorHydration<FeedbackInputs>({
+    schema: feedbackSchema,
+    formState,
+    onFoundError: handleFoundError,
+  })
+
+  // Watch & update the character count for the "Support summary" textarea
+  const suggestions = watch('suggestions') || ''
+  const remainingCharacters =
+    suggestions.length >= TEXTAREA_MAX_CHARACTERS ? 0 : TEXTAREA_MAX_CHARACTERS - suggestions.length
+
+  const { defaultValues } = formState
+
+  return (
+    <>
+      <NextSeo
+        title={`Give your feedback - Find, Recruit and Follow-up`}
+        description="Let us know what you think of our new Find, Recruit and Follow-up website. Your feedback will help us to improve it."
+      />
```

Comment:
we are still using the feedback page title and description, this is for seo so let's update it to match the new page